src/collective/cover/tiles/carousel.py

```diff
@@ -66,7 +65,10 @@ def populate_with_object(self, obj):
         :type uuids: List of strings
         """
         super(ListTile, self).populate_with_object(obj)  # check permission
-        uuids = ICoverUIDsProvider(obj).getUIDs()
+        if obj.portal_type == 'Collection':
```

Comment:
I don't limit it because is ignored anyway as it has no image field, but you can activate the lead image behavior on a Dexterity-based folder and then the folder will be shown in the gallery. I'm against most "magic" because it only adds complexity to the code and we're not sure if someone is using it anyway.

populating a carousel with a folder could be a nice shortcut at first, but I doubt people use it on a daily base; populating a carousel with a collection on the other side still makes some sense. any way, in none of our projects this is being used; all editors want to keep absolute control of the composite page.